### DISCUSSION TITLE
请问可以后台运行吗

### DISCUSSION THREAD

**Discussion by 1475505** [taskwitch/txedu-helper#1] [Q&A] [answered]
就是A标签页为腾讯课堂，焦点在B标签页，A的自动签到功能是否依旧工作？

**Comment by taskwitch** [comment, marked as answer]
用户您好, 简短的回答是: **可以**.
脚本可以在页面不刷新的情况下在后台一直运行, 刷新后需要重新运行加载. 但是尚不明确浏览器或者是腾讯课堂页面是否会有自动刷新机制.(以作者测试来看没有)
如果您使用谷歌浏览器, 可以使用如下方法关闭**浏览器的**自动刷新:
> 1. 在浏览器地址栏输入`chrome://flags/#automatic-tab-discarding`回车
> 2. 将选项设置为disabled
> **如果找不到该选项说明浏览器版本不同, 请去搜索引擎查找解决方案! **
